src/main/java/domain/Lotto.java

```diff
@@ -0,0 +1,57 @@
+package domain;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class Lotto{
+
+    private static final int LOTTO_NUMBER_COUNT = 6;
+    private List<Integer> lottoNumbers;
```

Comment:
```suggestion
    private static final int LOTTO_NUMBER_COUNT = 6;

    private List<Integer> lottoNumbers;
```

P3) 보통 이렇게 상수랑 변수는 줄바꿈으로 구분하는 편이에요